### PR TITLE
Update Barco

### PR DESCRIPTION
**Pull request by DFFspace**
Also removed Projector Toolset as they updated it to: Not applicable

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -263,9 +263,8 @@ _Note: daily releases of this software list are listed, including CSV and JSON f
 |:----------------|:----------------|:--------------------:|:--------------------:|:---------------------:|:---------------------:|:---------------------:|:----------------|-----------:|
 | Backblaze      | Cloud | N/A (SaaS) | Not vuln | Fix |  | | Cloud service patched |[source](https://help.backblaze.com/hc/en-us/articles/4412580603419) |
 | Barco          | Demetra | |  | Investigation |  | | | [source](https://www.barco.com/en/support/knowledge-base/KB12495) |
-| Barco          | OpSpace | 1.8 | Not vuln | Workaround |  | | | [source](https://www.barco.com/en/support/knowledge-base/KB12493) |
+| Barco          | OpSpace | 1.8 | Not vuln | Fix | Vulnerable | Vulnerable | | [source](https://www.barco.com/en/support/knowledge-base/KB12493) |
```
Comment (tschmidtb51) [inline software/README.md:266]: According to my understanding the version string should be 1.8 - 1.9.4.1

Comment (DFFspace) [reply software/README.md]: Added 1.8 - 1.9.4.1

File: software/README.md [hunk 2]
```diff
@@ -263,9 +263,8 @@ _Note: daily releases of this software list are listed, including CSV and JSON f
 |:----------------|:----------------|:--------------------:|:--------------------:|:---------------------:|:---------------------:|:---------------------:|:----------------|-----------:|
 | Backblaze      | Cloud | N/A (SaaS) | Not vuln | Fix |  | | Cloud service patched |[source](https://help.backblaze.com/hc/en-us/articles/4412580603419) |
 | Barco          | Demetra | |  | Investigation |  | | | [source](https://www.barco.com/en/support/knowledge-base/KB12495) |
-| Barco          | OpSpace | 1.8 | Not vuln | Workaround |  | | | [source](https://www.barco.com/en/support/knowledge-base/KB12493) |
+| Barco          | OpSpace | 1.8 - 1.9.4.1 | Not vuln | Fix | Vulnerable | Vulnerable | | [source](https://www.barco.com/en/support/knowledge-base/KB12493) |
```
Comment (tschmidtb51) [inline software/README.md:266]:
```suggestion:
|Barco | OpSpace | 1.8 - 1.9.4.1 | Not vuln | Workaround | Vulnerable | Vulnerable | Hotfix available: R33054201 | [source](https://www.barco.com/en/support/knowledge-base/KB12493) |

**Comment by tschmidtb51** [comment]
Thank you for your contribution. Please also add [Demetra](https://www.barco.com/en/support/knowledge-base/KB12503) as currently under investigation.
Thank you.

**Comment by DFFspace** [comment]
Status updated for Demetra

**Pull request merged**